client/public/locales/en/translation.json

```diff
@@ -198,6 +198,8 @@
     "noDataAvailableTitle": "No data available",
     "noResultsFoundBody": "No results match the filter criteria. Remove all filters or clear all filters to show results.",
     "noResultsFoundTitle": "No results found",
+    "noCodesSnippetAvailableTitle": "No code snippet available",
+    "noCodesSnippetAvailableBody": "The code snippet for this incident is not available.",
```

Comment:
The messages makes it sound more like a code snippet was generated but not currently available to display.

Maybe the body message can read more like "No code snippet was created from this incident."

Reply:
Although now I'm reminded of this commercial: https://youtu.be/zNCrMEOqHpc